pkg/environment/environment.go

```diff
@@ -9,13 +9,17 @@ import (
 	"os"
 	"regexp"
 
+	"github.com/Azure/application-gateway-kubernetes-ingress/pkg/azure"
 	"github.com/golang/glog"
 )
 
 const (
 	// AzContextLocationVarName is an environment variable name. This file is available on azure cluster.
 	AzContextLocationVarName = "AZURE_CONTEXT_LOCATION"
 
+	// AzureEnvironmentVarName is an evironment variable name. It represents the cloud in which the application is running.
```

Comment:
It would be helpful to give a few examples of what this could be.

Reply:
I will remove this. I was experimenting with AzureEnvironment. Will move this out to another PR.